Composer/packages/electron-server/src/splash/splashScreen.ts

```diff
@@ -50,6 +50,7 @@ export const initSplashScreen = async ({
     webPreferences: {
       // This is necessary to enable loading local images in the url protocol (window.loadURL)
       webSecurity: false, // lgtm [js/disabling-electron-websecurity]
+      nativeWindowOpen: true,
```

Comment:
The `nativeWindowOpen: false` is deprecated since v14. Documentation suggests to switch.

Reply:
I'm going through the app currently to figure out the impact. Don't see the difference at this point

Reply:
Seems fine as well 👍

Reply:
I tested almost all `window.open` I found. We have our handler in Electron which processes them correctly, so looks like we're good to go